product_docs/docs/pgd/5/manual/05-creating-cluster.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Creating the PGD Cluster
+---
+
+So far, we have:
+
+* Created three Hosts.
+* Installed a Postgres server on each host.
+* Installed Postgres Distributed on each host.
+* Configured the Postgres server to work with PGD on each host.
+
+To create the cluster, we tell `host-one`'s Postgres instance that it is a PGD node - `node-one` and create PGD groups on that node.
+Then we tell `host-two` and `host-three`'s Postgres instances that they are PGD nodes - `node-two` and `node-three` and that they should join a group on `node-one`.
+
+To create a node, it must be given a name and a connection string which other nodes can use to connect to it.
+
+For each node we want to create a connection string which will allow PGD to preform replication.
+
+## Create connection strings for each node
+
+The connection string is a key/value string which starts with a `host=` and the IP address of the host (or if you have resolvable named hosts, the name of the host).
+That is followed by the name of the database; `dbname=bdrbd` as we created a `bdrdb` database when [installing the software](04-installing-software).
+
+Calculate the connection strings for each of your nodes in advance. 
+Below are the connection strings for our 3 node example:
+
+| Name       | Node Name  | Private IP      | Connection string              |
+| ---------- | ---------- | --------------- | ------------------------------ |
+| host-one   | node-one   | 192.168.254.166 | host=host-one dbname=bdrdb     |
+| host-two   | node-two   | 192.168.254.247 | host=host-two dbname=bdrdb     |
+| host-three | node-three | 192.167.254.135 | host=host-three dbname=bdrdb   |
+
+The process will involve:
+
+* Logging into host-one's Postgres server
+  * Running `bdr.create_node` to give it a node name (node-one) and connection string
+  * Creating a top-level group called `pgd` with `bdr.create_group`
+  * Creating a sub-group called `dc1` as a child of `pgd` with `bdr.create_group`
+* Logging into host-two's Postgres server
+  * Running `bdr.create_node` to give it a node name (node-two) and connection string
+  * Running `bdr.join_node_group` with node-one's connection string and group name `dc1`
+* Logging into host-three's Postgres server
+  * Running `bdr.create_node` to give it a node name (node-three) and connection string
+  * Running `bdr.join_node_group` with node-one's connection string and group name `dc1`
+
+
+## Preparing the first node 
+
+Log into host-one's Postgres server.
+
+```
+ssh rocky@host-one
+sudo -iu enterprisedb psql bdrdb
+```
+
+### Create the first node
+
```

Comment:
Should we suggest the use of a dedicated user? https://www.enterprisedb.com/docs/pgd/latest/security/roles/

Reply:
Dedicated user for what? Replication?

Reply:
yes, a PGD dedicated user that will take care of everything, relpication included.
But if it's only a quick start, then we can even use the superuser, maybe a hint on not using the superuser when they actually set up the prod env.

Reply:
Going to address dedicated users later. Want to work out a conistent way of presenting them without it being overly confusing with OS and PG users.